packages/serverless-orchestration/src/ServerlessHub.js

```diff
@@ -121,17 +121,16 @@ hub.post("/", async (req, res) => {
 
       // Store block number data for this chain ID which we'll use to update the GCP cache later.
       blockNumbersForChain[chainId] = {
-        lastQueriedBlockNumber,
-        latestBlockNumber,
+        lastQueriedBlockNumber: Number(lastQueriedBlockNumber),
```

Comment:
this type was sometimes a string when I tested it. thought it best to ensure it's a number. @nicholaspai let me know if you have a problem with this